bin/node/cli/Cargo.toml

```diff
@@ -101,9 +101,7 @@ try-runtime-cli = { version = "0.10.0-dev", optional = true, path = "../../../ut
 serde_json = "1.0.85"
 
 [target.'cfg(any(target_arch="x86_64", target_arch="aarch64"))'.dependencies]
-sp-trie = { version = "7.0.0", default-features = false, path = "../../../primitives/trie", features = [
-	"memory-tracker",
-] }
+sp-trie = { version = "7.0.0", default-features = false, path = "../../../primitives/trie" }
```

Comment:
The dependency isn't used as far as I can see. I think it was only here to enable the feature.

Reply:
Seems like it.